front-end/src/domains/game-board/components/game-grid/game-grid.utils.ts

```diff
@@ -0,0 +1,69 @@
+import { Colors, size } from "./game-grid.constant";
+import {
+  Grid,
+  GridItem,
+  LandType,
+  NodeType,
+  SpaceType,
+} from "./game-grid.types";
+
```

Comment:
For all the helper functions we need to render the UI, it makes sense to put all these in their own utils file.

That way the component file only holds the UI + composition 😎